examples/react-native/__tests__/ts/realm-database/change-an-object-model-test.tsx

```diff
@@ -105,49 +202,65 @@ describe('Change an Object Model Tests', () => {
         }
       },
     };
+
     // pass the configuration object with the updated 'schemaVersion' and 'migration' function to createRealmContext()
-    createRealmContext(config);
-    // :replace-end:
+    const {RealmProvider, useRealm} = createRealmContext(config);
     // :snippet-end:
+
+    const App = () => (
+      <RealmProvider>
+        <RestOfApp />
+      </RealmProvider>
+    );
+
+    const RestOfApp = () => {
+      const realm = useRealm();
+
+      // This test assumes only one object model exists.
+      expect(realm.schema[0]).not.toHaveProperty('properties.lastName');
+      expect(realm.schema[0]).not.toHaveProperty('properties.firstName');
+      expect(realm.schema[0]).toHaveProperty('properties.fullName');
+      expect(realm.schemaVersion).toBe(4);
+
+      return (
+        <View>
+          <Text>This is the rest of the app!</Text>
+        </View>
+      );
+    };
+
+    render(<App />);
   });
 
   it('should modify a property type', () => {
     // :snippet-start: modify-a-property-type
-    // :replace-start: {
-    //  "terms": {
-    //   "MyTask": "Task"
-    //   }
-    // }
-    class MyTask extends Realm.Object<MyTask> {
+
+    class Person extends Realm.Object<Person> {
       _id: Realm.BSON.ObjectId = new Realm.BSON.ObjectId();
```

Comment:
i think you shouldn't set a default value here as it currently does. the JS example doesn't have this.

```suggestion
      _id: Realm.BSON.ObjectId;
```